### PR TITLE
SONAR-10289 Update login page UI and allow to display identity providers help

### PR DESCRIPTION
**Pull request by stas-vilchik**
@gregaubert please review only the last commit

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/sessions/components/OAuthProviders.tsx [hunk 1]
```diff
@@ -41,7 +45,10 @@ export default function OAuthProviders(props: Props) {
                 `${getBaseUrl()}/sessions/init/${identityProvider.key}` +
                 `?return_to=${encodeURIComponent(props.returnTo)}`
               }
-              style={{ backgroundColor: identityProvider.backgroundColor }}
+              style={{
+                backgroundColor: identityProvider.backgroundColor,
+                color: getTextColor(identityProvider.backgroundColor, theme.secondFontColor)
+              }}
               title={formatLabel(identityProvider.name)}>
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/sessions/components/OAuthProviders.tsx]: I don't think this is needed, or a Tooltip should be used

File: server/sonar-web/src/main/js/apps/sessions/components/LoginFormContainer.tsx [hunk 2]
```diff
@@ -83,7 +84,9 @@ class LoginFormContainer extends React.PureComponent<Props, State> {
 
     return (
       <LoginForm
-        identityProviders={identityProviders}
+        identityProviders={identityProviders.map(
+          i => (i.key === 'github' ? { ...i, help: 'blablabla' } : i)
```
[the remark this answers is not in the document]
Comment (stas-vilchik) [reply server/sonar-web/src/main/js/apps/sessions/components/LoginFormContainer.tsx]: 😱

File: server/sonar-web/src/main/js/apps/sessions/components/EmailAlreadyExists.tsx [hunk 3]
```diff
@@ -96,41 +97,43 @@ export default class EmailAlreadyExists extends React.PureComponent<Props, State
...
-          <p className="little-spacer-bottom">
-            <FormattedMessage
-              defaultMessage={translate('sessions.email_already_exists.1')}
-              id="sessions.email_already_exists.1"
-              values={{ email: <strong>{query.email}</strong> }}
-            />
-          </p>
-          {this.renderIdentityProvier(query.existingProvider, query.existingLogin)}
-        </div>
+      <div id="bd" className="page-wrapper-simple">
```
[the remark this answers is not in the document]
Comment (stas-vilchik) [reply server/sonar-web/src/main/js/apps/sessions/components/EmailAlreadyExists.tsx]: well, at some point it was used in ITs, I can't find its usage anymore though...

File: server/sonar-web/src/main/js/apps/sessions/components/SimpleSessionsContainer.tsx [hunk 4]
```diff
@@ -18,12 +18,23 @@
...
+import GlobalFooterContainer from '../../../app/components/GlobalFooterContainer';
+import * as theme from '../../../app/theme';
+import NavBar from '../../../components/nav/NavBar';
 
 interface Props {
   children?: React.ReactNode;
 }
 
 export default function SimpleSessionsContainer({ children }: Props) {
-  return <SimpleContainer hideLoggedInInfo={true}>{children}</SimpleContainer>;
```
[the remark this answers is not in the document]
Comment (stas-vilchik) [reply server/sonar-web/src/main/js/apps/sessions/components/SimpleSessionsContainer.tsx]: good point

**Comment by stas-vilchik** [comment]
@gregaubert I got confused in my own development, so I had to force push 😄

**Comment by stas-vilchik** [comment]
@gregaubert also Laura confirmed this is fine to display the login form on the gray background

**Pull request merged**